src/main/java/io/supertokens/totp/Totp.java

```diff
@@ -241,7 +245,7 @@ private static void checkAndStoreCode(TenantIdentifierWithStorage tenantIdentifi
                         try {
                             totpSQLStorage.insertUsedCode_Transaction(con, tenantIdentifierWithStorage, newCode);
                             totpSQLStorage.commitTransaction(con);
-                        } catch (UsedCodeAlreadyExistsException | TotpNotEnabledException e) {
+                        } catch (UsedCodeAlreadyExistsException | UnknownDeviceException e) {
```

Comment:
See my comment in the plugin interface PR

Reply:
Done.